backend/src/schema/resolvers/reports.js

```diff
@@ -38,12 +29,11 @@ export default {
           },
         )
         log(reportTransactionResponse)
-        return reportTransactionResponse.records.map(transformReturnType)
+        return reportTransactionResponse.records.map(record => record.get('returnedResource'))
       })
       try {
-        const [createdRelationshipWithNestedAttributes] = await reportWriteTxResultPromise
-        if (!createdRelationshipWithNestedAttributes) return null
-        return createdRelationshipWithNestedAttributes
+        const [reportedResource] = await reportWriteTxResultPromise
+        return reportedResource || null
       } finally {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-18T16:07:13Z" title="Tuesday, February 18th 2020, 5:07:13 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -38,12 +29,12 @@ export default {
           },
         )
         log(reportTransactionResponse)
-        return reportTransactionResponse.records.map(transformReturnType)
+        return reportTransactionResponse.records.map(record => record.get('returnedResource'))
       })
       try {
-        const [createdRelationshipWithNestedAttributes] = await reportWriteTxResultPromise
-        if (!createdRelationshipWithNestedAttributes) return null
-        return createdRelationshipWithNestedAttributes
+        const [reportedResource] = await reportWriteTxResultPromise
+        if (!reportedResource) return null
+        return reportedResource
```


```suggestion
        return reportedResource || null
```
If you want to convert `undefined` to `null`.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-18T20:53:25Z" title="Tuesday, February 18th 2020, 9:53:25 pm +01:00">Feb 18, 2020</time>_

---


that makes sense!

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-19T07:40:07Z" title="Wednesday, February 19th 2020, 8:40:07 am +01:00">Feb 19, 2020</time>_

---


Yes, I like this better! 😍 @roschaefer